content/docs/user-guide/settings/settings-registry-developer-documentation.md

```diff
@@ -0,0 +1,1389 @@
+  * [Settings Registry Merge Utils Specialization API](#settings-registry-merge-utils-specialization-api)
+  * [Settings Registry Merge Utils Value Dumper API](#settings-registry-merge-utils-value-dumper-api)
+  * [Settings Registry Manifest Visit Utils API](#settings-registry-manifest-visit-utils-api)
+* [Setting Registry Specializations - Folder Merging](#setting-registry-specializations-folder-merging)
+    * [Ex. Loading \*.setreg files using Specialization Tags](#loading-setreg-using-specialization)
+  * [Adding Specializations via the Command Line](#adding-specializations-via-the-command-line)
+    * [Ex. Running the Editor with a custom specialization](#running-the-editor-with-custom-specialization)
+  * [Settings Registry Specialization System: MergeSettingsFolder()  - In-Depth Explanation](#settings-registry-specialization-system-in-depth)
+* [Settings Registry - Command Line Support](#settings-registry-command-line-support)
+  * [Command Line option order evaluation](#cli-option-order-evaluation)
+  * [Ex. Using "--regset" to set override the Project Path](#regset-cli-example)
+  * [Ex. Using "--regdump" to dump the JSON object containing the bootstrap settings to stdout](#regdump-cli-example)
+* [Loading Gems through the Settings Registry](#loading-gems-through-the-settings-registry)
+  * [Ex. Automatically loading the list of gem build dependencies for an Application](#automatic-gem-dependency-loading)
+    * [CMakeLists.txt (Voxel Editor)](#cmakeliststxt-voxel-editor)
+    * [CMakeLists.txt (AtomBridge)](#cmakeliststxt-atombridge)
+    * [cmake\_dependencies.voxeleditor.setreg (Voxel Editor)](#cmake-dependencies-voxel-editor)
+    * [CMakeLists.txt (AtomTest)](#cmakeliststxt-atom-test-project-gem)
+  * [Platform Gem Loading](#platform-gem-loading)
+* [Explicit Gem Activation](#explicit-gem-activation)
+  * [Ex. Activating a Gem for the AutomatedTesting Project ](#gem-dependency-automatedtesting)
+  * [Disable Autoloading of Gem Modules](#disable-autoloading-of-gem-modules)
+    * [Ex. Disabling specific Gem modules from autoloading when activated as part of the AutomatedTesting Project](#disable-autoloading-of-chatplay-module)
+    * [Disable Autoloading for a user](#disable-autoloading-at-the-user-level)
+    * [Disable Autoloading for a project](#disable-autoloading-at-the-project-level)
+    * [Disable Autoloading for a platform](#disable-autoloading-at-the-platform-level)
+  * [How to Load Gems in C++](#advanced-topic-load-gems-in-cpp)
+* [Add AssetProcessor Scan Folder for Gems using the Settings Registry](#assetprocessor-scan-folder-support)
+  * [Gem Asset Scan Folders](#gem-asset-scan-folders)
+    * [assetprocessor\_settings.setreg](#assetprocessorsettings-setreg)
+* [Convert Windows Style INI file to JSON format](#convert-windows-style-ini-file-to-json-format)
+* [Settings Registry - Debugging](#settings-registry-debugging)
+  * [MSVC Debugging](#msvc-debugging)
+
+
+
+<a id="WhatIsTheSettingsRegistry"></a>
```

Comment:
Don't need html anchors, markdown supports anchor like mechanisms

Reply:
While markdown does support anchor like mechanisms, the naming scheme of those anchors are ambigious between the different flavors of markdown.
Some markdown replaces non html support characters with dashes(i.e '?') and some require that to link to the implicit anchor made by a header tag `#`, `##`, `###` etc..., it requires prefixing the anchor with the name of the document